cellprofiler/modules/metadata.py

```diff
@@ -1164,6 +1177,19 @@ def on_setting_changed(self, setting, pipeline):
                 self.refresh_group_joiner(group)
 
     def update_table(self):
+        for group in self.extraction_methods:
+            if group.extraction_method == X_AUTOMATIC_EXTRACTION and not group.metadata_autoextracted:
+                    response = wx.MessageBox(
```

Comment:
Can you move the wx import down here, so that it only gets imported if it needs to be?

Reply:
@bethac07 It was originally also imported by do_update_metadata() as well, so I'd need to restore that too. Wasn't sure if it's better practice to just import it with the module or to load wx each time you hit the update button/do an extraction?